### PR TITLE
Adding ru-ru locales

### PR DESCRIPTION
**Pull request by ros-tel**

### CODE REVIEW

**Comment by markjcrane** [comment]
Please make sure all contributor sign up as a contributor with their github accounts so that I can accept the translation. 

I studied Russian for a year in college for fun. Very rusty now so I couldn't have done it. So I'm excited about the translation. I know it was a lot of work to make it. Good job.

File: core/users/app_languages.php [hunk 1]
```diff
@@ -152,6 +164,7 @@
 $text['option-available']['pt-pt'] = "Disponível";
 $text['option-available']['fr-fr'] = "Disponible";
 $text['option-available']['pl'] = "Dostępny";
+$text['option-available']['ru-ru'] = "Доступный";
```
Comment (moteus) [inline core/users/app_languages.php]: Доступнен
image

File: app/sip_status/app_languages.php [hunk 2]
```diff
@@ -87,6 +94,7 @@
 $text['header-sip-status']['uk'] = "Стан SIP";
 $text['header-sip-status']['de-at'] = "SIP Status";
 $text['header-sip-status']['it-it'] = "Stato SIP";
+$text['header-sip-status']['ru-ru'] = "SIP статус";
```
Comment (moteus) [inline app/sip_status/app_languages.php]: `Статус SIP` как в пункте меню

File: app/sip_status/app_languages.php [hunk 3]
```diff
@@ -153,6 +166,7 @@
 $text['button-flush_registrations']['uk'] = "Скинути реєстрації";
 $text['button-flush_registrations']['de-at'] = "Registrierungen verwerfen";
 $text['button-flush_registrations']['it-it'] = "Aggiorna registrazioni";
+$text['button-flush_registrations']['ru-ru'] = "Отключить Регистрацию";
```
Comment (moteus) [inline app/sip_status/app_languages.php]: Сбросить регистрацию

File: core/menu/app_menu.php [hunk 4]
```diff
@@ -6,6 +6,7 @@
 $apps[$x]['menu'][0]['title']['pt-pt'] = "Gestor de Menus";
 $apps[$x]['menu'][0]['title']['pt-br'] = "Menu Admistrador";
 $apps[$x]['menu'][0]['title']['pl'] = "Menedżer menu";
+$apps[$x]['menu'][0]['title']['ru-ru'] = "Меню управления";
```
Comment (moteus) [inline core/menu/app_menu.php]: Управление меню
Редактор меню

File: core/menu/app_languages.php [hunk 5]
```diff
@@ -186,6 +201,7 @@
 $text['label-menu_order']['fr-fr'] = "Ordre";
 $text['label-menu_order']['pt-br'] = "Ordem";
 $text['label-menu_order']['pl'] = "Kolejność";
+$text['label-menu_order']['ru-ru'] = "Порядковый номер";
```
Comment (moteus) [inline core/menu/app_languages.php]: Приоритет

File: core/menu/app_languages.php [hunk 6]
```diff
@@ -138,6 +149,7 @@
 $text['label-title']['fr-fr'] = "Titre";
 $text['label-title']['pt-br'] = "Título";
 $text['label-title']['pl'] = "Tytuł";
+$text['label-title']['ru-ru'] = "Заглавие";
```
Comment (moteus) [inline core/menu/app_languages.php]: Название

File: app/dialplan/app_languages.php [hunk 7]
```diff
@@ -701,6 +759,7 @@
 $text['label-data']['de-de'] = "Daten";
 $text['label-data']['de-at'] = "Daten";
 $text['label-data']['he'] = "מידע";
+$text['label-data']['ru-ru'] = "Дата";
```
Comment (moteus) [inline app/dialplan/app_languages.php]: Данные, Аргумент
image

Но может не стоит это переводить? `inline`/`break-on`,

File: core/users/app_languages.php (hunk 1 above)
Comment (ros-tel) [inline core/users/app_languages.php]: fixed with https://github.com/fusionpbx/fusionpbx/pull/2530/commits/6733031b00113df21db37373d2bdc65dd27ca3f9

File: app/sip_status/app_languages.php (hunk 2 above)
Comment (ros-tel) [inline app/sip_status/app_languages.php]: fixed with https://github.com/fusionpbx/fusionpbx/pull/2530/commits/6733031b00113df21db37373d2bdc65dd27ca3f9

File: app/sip_status/app_languages.php (hunk 3 above)
Comment (ros-tel) [inline app/sip_status/app_languages.php]: fixed with https://github.com/fusionpbx/fusionpbx/pull/2530/commits/6733031b00113df21db37373d2bdc65dd27ca3f9

File: core/menu/app_menu.php (hunk 4 above)
Comment (ros-tel) [inline core/menu/app_menu.php]: fixed with https://github.com/fusionpbx/fusionpbx/pull/2530/commits/6733031b00113df21db37373d2bdc65dd27ca3f9

File: core/menu/app_languages.php (hunk 5 above)
Comment (ros-tel) [inline core/menu/app_languages.php]: fixed with https://github.com/fusionpbx/fusionpbx/pull/2530/commits/6733031b00113df21db37373d2bdc65dd27ca3f9

File: core/menu/app_languages.php (hunk 6 above)
Comment (ros-tel) [inline core/menu/app_languages.php]: fixed with https://github.com/fusionpbx/fusionpbx/pull/2530/commits/6733031b00113df21db37373d2bdc65dd27ca3f9

File: app/dialplan/app_languages.php (hunk 7 above)
Comment (ros-tel) [inline app/dialplan/app_languages.php]: Аргумент, inline, break https://github.com/fusionpbx/fusionpbx/pull/2530/commits/6733031b00113df21db37373d2bdc65dd27ca3f9

File: app/operator_panel/app_languages.php [hunk 8]
```diff
@@ -219,6 +235,7 @@
 $text['label-call_group']['de-at'] = "Rufgruppe";
 $text['label-call_group']['he'] = "קבוצת חיוג";
 $text['label-call_group']['it-it'] = "Chiamata di Gruppo";
+$text['label-call_group']['ru-ru'] = "Вызов Группы";
```
Comment (moteus) [inline app/operator_panel/app_languages.php]: Группы вызова

Comment (ros-tel) [inline app/operator_panel/app_languages.php]: fixid https://github.com/fusionpbx/fusionpbx/pull/2530/commits/9cf6947b586c9fa9b46223824be0c32507c3e164

**Comment by ros-tel** [comment]
All signed CLA
https://github.com/fusionpbx/open-source/pull/86
https://github.com/fusionpbx/open-source/pull/87
https://github.com/fusionpbx/open-source/pull/88
https://github.com/fusionpbx/open-source/pull/89

**Comment by markjcrane** [comment]
Thanks ros-tel and all those that have shared your translations and work to add Russian to FusionPBX. +1

**Pull request merged**